examples/notebooks/get-started/try-apache-beam-yaml.ipynb

```diff
@@ -0,0 +1,671 @@
+    "16,Elie,10,Italy,doctor\n",
+    "17,Shaylyn,34,Belgium,worker\n",
+    "18,Fayre,33,Spain,police officer\n",
+    "19,Sabina,52,Germany,police officer\n",
+    "20,Aryn,20,Belgium,police officer\n",
+    "21,Darlleen,49,Spain,worker\n",
+    "22,Jere,18,Italy,worker\n",
+    "23,Candi,60,Germany,police officer\n",
+    "24,Sindee,40,Germany,firefighter\n",
+    "25,Selma,20,Spain,worker\n",
+    "26,Vonny,35,Germany,doctor\n",
+    "27,Kate,53,Spain,worker\n",
+    "28,Annabela,48,Belgium,worker\n",
+    "29,Jenilee,55,Germany,police officer\n",
+    "30,Gusella,44,France,police officer\n",
+    "31,Fawne,35,Spain,worker\n",
+    "32,Karolina,39,Spain,police officer\n",
+    "33,Sadie,58,Germany,firefighter\n",
+    "34,Clo,10,Italy,police officer\n",
+    "35,Beth,46,Spain,firefighter\n",
+    "36,Adore,18,Italy,firefighter\n",
+    "37,Tarra,29,Spain,doctor\n",
+    "38,Jessamyn,36,France,police officer\n",
+    "39,Deedee,24,Germany,unemployed\n",
+    "40,Patricia,45,Italy,doctor\n",
+    "41,Wileen,39,Spain,police officer\n",
+    "42,Paola,55,Italy,worker\n",
+    "43,Gwyneth,37,Italy,worker\n",
+    "44,Stacey,36,Spain,worker\n",
+    "45,Camile,60,Germany,unemployed\n",
+    "46,Sheree,10,Spain,unemployed\n",
+    "47,Albertina,53,France,police officer\n",
+    "48,Jinny,30,Spain,firefighter\n",
+    "49,Kayla,57,Italy,firefighter\n",
+    "50,Jaime,55,France,doctor\n",
+    "'''\n",
+    "\n",
+    "save_to_file(csv_data, 'data/people.csv')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's validate if the file was created correctly. You should see the first few lines from the generated file. Validate if the beginning of the file matches with the declared content above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! head data/people.csv"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Your first YAML pipelines\n",
+    "\n",
+    "In this section we'll present you the basic structure of a YAML pipeline and present you some available transforms.\n",
+    "Below is a simple pipeline that reads data from the csv file we've just created and logs the elements for debugging purposes.\n",
+    "\n",
+    "The `LogForTesting` transform lets us log the data when developing a pipeline. Remember, it is not advised to use this transform in production.\n",
+    "\n",
+    "Let's define the pipeline and save it to a file:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline = '''\n",
+    "pipeline:\n",
+    "  type: chain\n",
+    "  transforms:\n",
+    "    - type: ReadFromCsv\n",
+    "      config:\n",
+    "        path: data/people.csv\n",
+    "    - type: LogForTesting\n",
+    "'''\n",
+    "save_to_file(pipeline, 'pipelines/pipeline-01.yaml')"
```

Comment:
Colab has a built-in writefile method that I think is less messy and allows makes specifying string literals in YAML easier
```suggestion
    "%%writefile pipelines/pipeline-01.yaml\n",
    "\n",
    "pipeline:\n",
    "  type: chain\n",
    "  transforms:\n",
    "    - type: ReadFromCsv\n",
    "      config:\n",
    "        path: data/people.csv\n",
    "    - type: LogForTesting\n"
```